lib/ts/recipe/session/types.ts

```diff
@@ -189,6 +204,12 @@ export type RecipeInterface = {
         userContext: any;
     }): Promise<SessionContainerInterface>;
 
+    getGlobalClaimValidators(input: {
+        userId: string;
+        defaultClaimValidators: SessionClaimValidator[];
```

Comment:
I do not like the prefix `default`. We had decided that you do not want to call these global, but we need to think of another term